openrtb_ext/response.go

```diff
@@ -55,10 +55,11 @@ type ExtBidderError struct {
 
 // ExtHttpCall defines the contract for a bidresponse.ext.debug.httpcalls.{bidder}[i]
 type ExtHttpCall struct {
-	Uri          string `json:"uri"`
-	RequestBody  string `json:"requestbody"`
-	ResponseBody string `json:"responsebody"`
-	Status       int    `json:"status"`
+	Uri            string              `json:"uri"`
+	RequestBody    string              `json:"requestbody"`
+	ResponseBody   string              `json:"responsebody"`
+	Status         int                 `json:"status"`
+	RequestHeaders map[string][]string `json:"requestheaders"`
```

Comment:
The new parameter **RequestHeaders** will contain the header key and multiple values associated with it.
We have kept value as an array of strings because one header can have multiple values.
Also, this data structure is inline with [http/Header](https://golang.org/pkg/net/http/#Header)